tests/unit-tests/test_utils.py

```diff
@@ -658,6 +631,60 @@ def test_save_embeddings_with_id_mapping(num_embs, backend):
         assert len(saved_emb) == len(embs['n2'])
         assert_equal(embs['n2'][nid_mappings['n2']].numpy(), saved_emb.numpy())
 
+def helper_save_multiple_embeddings(tmpdirname):
+    emb_length0 = 103
+    emb_length1 = 205
+    nid_length = 12
```

Comment:
12 is actually dimension size.
You can take a look at https://pytorch.org/docs/main/generated/torch.rand.html. The first argument is *size, which can be a tuple defining the size of each dimension. The first dimension we usually call the length of the tensor (like number of raws). The second dimension is usually called the dimension size of the embedding.